### PR TITLE
Release web track 2 module

### PR DESCRIPTION
**Pull request by ArcturusZhang**
Pending on PR https://github.com/Azure/azure-rest-api-specs/pull/15088

- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

File: sdk/web/armweb/ci.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    include:
+    - sdk/web/armweb/
+
+pr:
+  paths:
+    include:
+    - sdk/web/armweb/
+
+stages:
+- template: ../../../eng/pipelines/templates/jobs/archetype-sdk-client-samples.yml
```
Comment (weshaggard) [inline sdk/web/armweb/ci.yml:13]:
```suggestion
- template: /eng/pipelines/templates/jobs/archetype-sdk-client-samples.yml
```

Comment (ArcturusZhang) [reply sdk/web/armweb/ci.yml]: This has been fixed on main

**Pull request closed without merging**